### ISSUE TITLE
Update to Ember-select

### ISSUE THREAD

**Issue by Saskapallayova** [appuniversum/ember-appuniversum#290]
During user testing for a new feature, we have noticed the users don't interact with Ember select the way we expect them to.

The user writes something in the text field, options show up in a drop down, but the user doesn't always select an option. 
e.g. if the user writes "Jacques" and they know they've written it correctly, they are not going to want to select a specific option, they expect what they've written to stay there.

Possible solution: if the user doesn't select an option from the dropdown, whatever the user writes in the text field should automatically be what's selected? 

@sofiemercy can give you access to the sessions if you need to see the interaction,

Timestamps are: 
Session 2 - 16:47
Session 4 - 5:45, 15:50
Session 5 - 7:50

**Comment by Windvis** [comment]
I've updated the title to make it more clear what this is about 😄.

TL;DR vanilla PowerSelect components aren't always the most useful for "searchbar" style components. We should provide an alternative if there is demand for this in other apps. 

We should analyze what the requirements for it are (and maybe create a design for it as well)?
